packages/react/src/components/ListItem/index.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import { Icon } from "..";
+import { ICONS } from "../../constants/";
+
+const ListItem = ({
+  active,
+  title,
+  subtitle,
+  subitem,
+  miniitem,
+  value,
+  leftIcon,
+  rightIcon,
+  hideDivider,
+  onClick,
+}) => (
+  <div
+    onClick={onClick}
+    className={cx({
+      ListItem: true,
+      "ListItem--subitem": subitem && !miniitem,
+      "ListItem--miniitem": miniitem,
+      "is-active": active,
+      "is-clickable": !!onClick,
+      "ListItem-divider": !hideDivider,
+    })}
+  >
+    {leftIcon && (
+      <div className="ListItem-leftIcon">
+        <Icon icon={leftIcon} />
+      </div>
+    )}
+    <div>
+      <div className="ListItem-title">{title}</div>
+      {!!subtitle && <div className="ListItem-subtitle">{subtitle}</div>}
+    </div>
+    <div className="ListItem-valueContainer">
+      <span>{value}</span>
+      <span className="ListItem-itemIcon">
+        {(onClick || rightIcon) && (
```

Comment:
Shouldn't be the `onClick` be on the main parent?
So we can have a larger clickable area?

Reply:
The `onClick` itself is bind to the main top `<div>` - indeed
Here it's only to check if `onClick` is there. If so - it's displaying the `>` icon

Reply:
Oh gotcha 👍